### DISCUSSION TITLE
"extensive usage will be throttled"

### DISCUSSION THREAD

**Discussion by nienienienie** [komoot/photon#822] [Q&A]
what does extensive usage exactly is?

https://photon.komoot.io/

what volume is extensive?

**Comment by lonvia** [comment]
There is no hard definition on "extensive usage". We throttle usage as required to protect the availability of the service for all. Putting a search box on your home page is unlikely to cause any problems. If you have large batches of addresses to geocode, you want to look into running your own instance of Photon.
